app/impact-reports.server.ts

```diff
@@ -18,8 +18,9 @@ let hypercertClient: HypercertClient | null = null;
 export const fetchReports = async (ownerAddress: string): Promise<Report[]> => {
 	try {
 		if (reports) {
-			console.log("Reports already exist, no need to fetch from remote");
-			console.log(`Existing reports: ${reports.length}`);
+			console.log(
+				`[📃] --REPORTS-- ${reports.length} already cached, no need to fetch from remote`,
+			);
```

Comment:
Remove console.log unless you are intentionally keeping it or add a comment so we know this log should stay to test caching.
```suggestion
```

Reply:
thanks! this is to show that the server side cache works, I think it can stay for now. I'm also not done with this PR as Im investigating rerenders